### PR TITLE
Handle the /usr/local/bin symlinks correctly on Mac 10.11.

### PR DESCRIPTION
**Pull request by sersut**

### CODE REVIEW

File: jenkins/verify-chef.sh [hunk 1]
```diff
@@ -10,24 +10,33 @@ export PATH
 EMBEDDED_BIN_DIR=/opt/$PROJECT_NAME/embedded/bin
 export EMBEDDED_BIN_DIR
 
+# If we are on Mac 10.11 our symlinks are located under /usr/local/bin
+# otherwise they are under /usr/bin
+if [[ $(uname -v) =~ ^Darwin\ Kernel\ Version\ 15 ]]; then
```
Comment (schisamo) [inline jenkins/verify-chef.sh]: We can simply this:

```bash
if test -f "/usr/bin/sw_vers"; then
```

Totally lifted from [Omnitruck's install.sh](https://github.com/chef/opscode-omnitruck/blob/84b9004fc8de26f5daff869400fffc26383d95d0/views/install.sh.erb#L139)

**Comment by schisamo** [comment]
We are also going to need to apply a similar fix to ChefDK's verify command:
https://github.com/chef/chef-dk/blob/443061eef820c01aad355a000b4200a9a0b4570d/lib/chef-dk/command/verify.rb#L239-L273

**Comment by sersut** [comment]
This fixes the issue for chef and derivative projects. 

For Chef DK we are only launching `chef verify` so the fix will be in chef-dk.

**Comment by schisamo** [comment]
:+1:

**Comment by scotthain** [comment]
:+1:

**Comment by yzl** [comment]
:+1:

File: jenkins/verify-chef.sh [hunk 2]
```diff
@@ -10,24 +10,39 @@ export PATH
 EMBEDDED_BIN_DIR=/opt/$PROJECT_NAME/embedded/bin
 export EMBEDDED_BIN_DIR
 
+# If we are on Mac 10.11 our symlinks are located under /usr/local/bin
+# otherwise they are under /usr/bin
+if [ -a /usr/bin/sw_vers ]; then
+  platform_version=`sw_vers | awk '/^ProductVersion:/ { print $2 }' | cut -d. -f1,2`
+
+  if [ $platform_version == "10.11" ]; then
```
Comment (schisamo) [inline jenkins/verify-chef.sh]: The use of `/usr/local/bin` should be true for all versions of Mac OS X now:
https://github.com/chef/omnibus-chef/blob/7d655f62dbb71c556002a9c1c4693e0edc1aed8a/package-scripts/chef/postinst#L36-L38

**Pull request merged**